src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -18,7 +20,11 @@ namespace Dynamo.WorkspaceDependency
     public class WorkspaceDependencyViewExtension : IViewExtension, ILogSource
     {
         internal MenuItem workspaceReferencesMenuItem;
-        private const String extensionName = "Workspace References";
+        private const string extensionName = "Workspace References";
+        private const string pythonPackage = "DSIronPython_Test";
+        private readonly Version pythonPackageVersion = new Version(1, 0, 7);
```

Comment:
Can we download the latest version of a package from PM by default because currently, AFAIK, we have to specify the exact version? Or better yet, can we just publish a 1.0.0 version of the IronPython extension package to prod?

Reply:
@aparajit-pratap I think the workspace references design is based on package name and version.. I do not anticipate user need to resolve a case later when they need to upgrade to a newer version. But who knows what could happen :) I would vote for being consistent with the existing behavior of other WS references

Reply:
@QilongTang I'm not sure I understand what you mean by staying consistent in this context?

Reply:
By "consistent" I mean WS Reference will display the package graph is depending on including name and version, if user local version is behind, user get a chance to choose to upgrade or not. Since you mentioned we always install the latest version, I want to make sure it is a consistent with what we have or not..

Reply:
I asked here if we can simply install the latest version that is available on the PM as an exception for the IronPython packages as otherwise, we'll have to hardcode this value in the code. I hope it is a one-time thing as I don't anticipate this package being updated in the near future.

Reply:
I don' t think there is an API to download the latest version, but there are APIs for getting the versions of a package, and then we could download the latest.

@zeusongit can you advise here?

Reply:
Yes, correct! We don't have an API that returns the latest version of a package directly, we may have to get the latest version from a separate request.
Although, as aparajit suggested, if it is a one-time thing, then we can just fetch the latest package by using the fixed version number.

But in case, we really don't want to hard code things here, we can think about adding a new API to the PM.